onmt/inputters/inputter.py

```diff
@@ -166,15 +171,6 @@ def make_features(batch, side):
     else:
```

Comment:
I'd advise against it. It's 6 lines of code that would need inlined in 6 places: trainer.py ``Trainer._gradient_accumulation`` twice, ``Trainer.validate`` twice, and then translator.py ``Translator._run_encoder`` once and ``Translator._score_target``.

Reply:
Calling `make_features(batch, 'src')` is equivalent to this one-liner:
```
src, lengths = batch.src if isinstance(batch.src, tuple) else batch.src, None
```
Calling  `make_features(batch, 'tgt')` is equivalent to this one-liner:

```
tgt = batch.tgt
```

This is much less than six lines of code. In either case, I think both one-liners are clearer than calling `make_features`. Even if the function is kept, it should not be called `make_features` because that isn't what it does.

Reply:
You're right! I got rid of ``make_features`` with those one liners. Thanks